### PR TITLE
fix: use hew Alert [WEB-1918]

### PR DESCRIPTION
**Pull request by johnkim-det**
## Description

We previously consolidated "Alerts" and "Messages", but we decided to revert that, since some cases may call for a smaller banner "Alert" vs. a larger format "Message"

## Test Plan

- Archive a model
- View model details page, make sure the archived status message is an `Alert` (horizontal, color, smaller):

Screen Shot 2024-01-18 at 11 11 06 AM

and NOT a `Message` (vertical, no color, larger):
Screen Shot 2024-01-18 at 11 12 18 AM

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

WEB-1918

### CODE REVIEW

File: webui/react/src/pages/F_ExpList/ComparisonView.tsx [hunk 1]
```diff
@@ -94,10 +95,9 @@ const ComparisonView: React.FC<Props> = ({
 
   const rightPane =
     selectedExperiments.length === 0 ? (
-      <Message
+      <Alert
         description="Select experiments you would like to compare."
-        icon="warning"
-        title="No experiments selected."
+        message="No experiments selected."
```
Comment (mapmeld) [inline webui/react/src/pages/F_ExpList/ComparisonView.tsx:100]: On other sections `icon="warning"` is replaced with `type="warning"` and would be added here for a 1:1 replacement

This is a large panel which I'd be OK to contain a Message or to change the intensity of the message. Maybe check in on Slack, we have been discussing how some experiments may be selected on an unloaded section so the wording is misleading.

Comment (johnkim-det) [reply webui/react/src/pages/F_ExpList/ComparisonView.tsx:100]: I think it used to be an `Alert` without a type, and I just added `icon="warning"` when I changed it to a `Message`.

If you think Message with warning icon is most appropriate for this, I can go ahead and revert the changes for this file.

Comment (mapmeld) [reply webui/react/src/pages/F_ExpList/ComparisonView.tsx:100]: ok that makes sense if it's going back to the previous component + props

**Review by mapmeld — APPROVED** [review]
✅  Confirmed that Alert replaces Message in these locations

**Pull request merged**